<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>901</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>391</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/social-card (1).jpg</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>120</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>ClickHere</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_1">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>251</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/wGuh9Zxxn0g (2).jpg</pixmap>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>281</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>SignIn</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_login">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>301</width>
       <height>341</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnAdd">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="leCity">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>180</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>City</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lbCity">
      <property name="geometry">
       <rect>
        <x>71</x>
        <y>180</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDel">
      <property name="geometry">
       <rect>
        <x>211</x>
        <y>300</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lbAbbr">
      <property name="geometry">
       <rect>
        <x>61</x>
        <y>10</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="leAbbr">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>6</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Abbr</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lbTitle">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>60</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="leInn">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>230</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>INN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lbInn">
      <property name="geometry">
       <rect>
        <x>71</x>
        <y>230</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEdit">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>300</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QLabel" name="teTitle">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>90</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnSelectAll">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>30</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Select All </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnect">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="teResult">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>409</y>
       <width>851</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="twOrg">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>521</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
